langmeta/langmeta/shared/src/main/scala/org/langmeta/internal/semanticdb/schema/LegacySemanticdb.scala

```diff
@@ -0,0 +1,137 @@
+package org.langmeta.internal.semanticdb.schema
+
+import org.langmeta.inputs
+
+import org.{langmeta => d}
+import scala.meta.internal.{semanticdb3 => s}
+import scala.meta.internal.semanticdb3.SymbolInformation.{Kind => k}
+import scala.meta.internal.semanticdb3.SymbolInformation.{Property => p}
+
+object LegacySemanticdb {
+
+  def toTextDocuments(database: Database): s.TextDocuments = {
+    val documents = database.documents.map(toTextDocument)
+    s.TextDocuments(documents)
+  }
+
+  def toTextDocument(document: Document): s.TextDocument = {
+
+    val input = inputs.Input.VirtualFile(document.filename, document.contents)
+
+    def toRange(pos: Position): s.Range = {
+      val r = inputs.Position.Range(input, pos.start, pos.end)
+      s.Range(r.startLine, r.startColumn, r.endLine, r.endColumn)
+    }
+
+    def toSymbol(symbol: String) = {
+      if (symbol.startsWith("_root_")) symbol
+      else if (symbol.startsWith("_empty_")) symbol
+      else if (symbol.startsWith("_star_")) symbol
+      // Convert old-style local symbol to new-style local symbol.
+      else "local" + symbol.replace('/', '_').replace('.', '_').replace('@', '_')
```

Comment:
How about we append an underscore to `"local"`, as in `"local_"`? I think this will make it symbols easier to read.